shared/actions/login/index.js

```diff
@@ -74,7 +74,7 @@ function getAccounts (): AsyncAction {
 
 export function login (): AsyncAction {
   return (dispatch, getState) => {
-    startLoginFlow(dispatch, getState, enums.provisionUi.ProvisionMethod.device, 'Keybase username', 'lorem ipsum', Constants.loginDone)
+    startLoginFlow(dispatch, getState, Constants.loginDone)
```

Comment:
i'm just killing this function. it used to be shared but now its unique so i'm just going to move it up